vm/devices/user_driver/src/vfio.rs

```diff
@@ -36,7 +36,7 @@ use zerocopy::AsBytes;
 use zerocopy::FromBytes;
 
 pub trait VfioDmaBuffer: 'static + Send + Sync {
-    /// Create a new dma buffer with the given `len` in bytes.
+    /// Create a new dma buffer with the given `len` in bytes. This buffer must be zeroed out.
```

Comment:
Nit, if you come back for another round, suggest:

/// Create a new DMA buffer of the given `len` bytes. Guaranteed to be zero-initialized.